### PR TITLE
Toggles to flip flow map channels

### PR DESCRIPTION
**Pull request by huwb**
To compensate for flow painting tools using different spaces

### CODE REVIEW

**Review by daleeidd — CHANGES_REQUESTED** [review]
I started this review ages ago and forgot to finish it. Sorry.

File: crest/Assets/Crest/Crest/Shaders/OceanInputs/FlowAddFlowMap.shader [hunk 1]
```diff
@@ -21,6 +23,9 @@ Shader "Crest/Inputs/Flow/Add Flow Map"
 			#pragma vertex Vert
 			#pragma fragment Frag
 
+			#pragma shader_feature_local _FLIPX_ON
+			#pragma shader_feature_local _FLIPZ_ON
```
[the remark this answers is not in the document]
Comment (huwb) [reply crest/Assets/Crest/Crest/Shaders/OceanInputs/FlowAddFlowMap.shader:27]: thx. could do. would that produce less variants? it seems like both approaches produce 4.

[the remark this answers is not in the document]
Comment (daleeidd) [reply crest/Assets/Crest/Crest/Shaders/OceanInputs/FlowAddFlowMap.shader:27]: You're right. I must have misinterpreted the documentation. Please ignore.

**Review by daleeidd — APPROVED** [review]

**Comment by huwb** [comment]
Cheers

**Pull request merged**